cache/src/main/java/com/playone/mobile/cache/BufferooCacheImpl.kt

```diff
@@ -26,7 +26,7 @@ class BufferooCacheImpl @Inject constructor(dbOpenHelper: DbOpenHelper,
 
     private val EXPIRATION_TIME = (60 * 10 * 1000).toLong()
 
-    private var database: SQLiteDatabase = dbOpenHelper.writableDatabase
+    private var database = dbOpenHelper.writableDatabase
```

Comment:
I think I can accept this coding style here but how about if the `writableDatabase` is return sub class of `SQLiteDatabase` and want to just present `database` as the super class?

Reply:
![jietu20180216-30254 pm](https://user-images.githubusercontent.com/5198104/36295374-96fc3d6c-132a-11e8-8640-7cb093e19e6b.png)

Here should be good! 👍